denizen_scripts/HeroCraft/items/tools/amethyst_tools/amethyst_hoe.dsc

```diff
@@ -6,18 +6,18 @@ Amethyst_hoe:
       max: 400
       current: 0
   data:
-    recipe_book_category: tools.Amethyst.hoe
+    recipe_book_category: tools.amethyst.hoe
   mechanisms:
     custom_model_data: 100
   lore:
-    - Far more brittle than diamond.
+    - <&r>Far more brittle than diamond.
   display name: <&f>Amethyst Hoe
   recipes:
     1:
       type: shaped
       output_quantity: 1
       input:
-      - Amethyst_ingot|Amethyst_ingot|air
+      - air|amethyst_ingot|amethyst_ingot
```

Comment:
Does not match vanilla recipe

Reply:
This should actually be a second recipe, not overwriting the first

Reply:
Will fix